String.cfc

```diff
@@ -1,5 +1,5 @@
 component {
-	string function escapeJSONcharacters(required string data) {
+	string function encodeForJSON(required string data) {
```

Comment:
i think ```data``` is a holdout from your initial pass. i think the argument should/could be ```required string string``` or ```required string value``` (i prefer ```string``` but maybe let someone else chime in)